api/tests/routes/native/v1/bookings_test.py

```diff
@@ -1103,6 +1105,26 @@ def test_return_venue_public_name(self, client):
         assert response.status_code == 200
         assert response.json["ongoing_bookings"][0]["stock"]["offer"]["venue"]["name"] == "Public name"
 
+    def test_return_venue_banner_url(self, client):
+        venue = offerers_factories.VenueFactory(bannerUrl="http://bannerUrl.com")
+        booking = booking_factories.BookingFactory(stock__offer__venue=venue)
+
+        client = client.with_token(booking.user.email)
+        response = client.get("/native/v1/bookings")
+
+        assert response.status_code == 200
+        assert response.json["ongoing_bookings"][0]["stock"]["offer"]["venue"]["bannerUrl"] == "http://bannerUrl.com"
+
+    def test_return_venue_is_open_to_public(self, client):
+        venue = offerers_factories.VenueFactory(isOpenToPublic=True)
+        booking = booking_factories.BookingFactory(stock__offer__venue=venue)
+
+        client = client.with_token(booking.user.email)
+        response = client.get("/native/v1/bookings")
+
+        assert response.status_code == 200
+        assert response.json["ongoing_bookings"][0]["stock"]["offer"]["venue"]["isOpenToPublic"] == True
```

Comment:
Détail, par homogénéité du code on utilise `is True` ou `is False`.
```suggestion
        assert response.json["ongoing_bookings"][0]["stock"]["offer"]["venue"]["isOpenToPublic"] is True
```